lua/modes.lua

```diff
@@ -222,8 +227,8 @@ M.enable_managed_ui = function()
 
 		if config.set_cursor then
 			vim.opt.guicursor:append('v-sm:ModesVisual')
-			vim.opt.guicursor:append('i-ci-ve:ModesInsert')
-			vim.opt.guicursor:append('r-cr-o:ModesOperator')
+			vim.opt.guicursor:append('i-ve:ModesInsert')
+			vim.opt.guicursor:append('r-o:ModesOperator')
```

Comment:
Was this `guicursor` change intentional? It doesn't seem to be related to the purpose of this PR

Reply:
Let's separate the cmdline highlight issue into its own PR then